pkg/kubelet/kubelet_test.go

```diff
@@ -2422,6 +2426,10 @@ func TestSyncLabels(t *testing.T) {
 					} else {
 						savedNode = test.existingNode
 					}
+					if savedNode.Labels == nil {
```

Comment:
```suggestion
					if savedNode == nil || savedNode.Labels == nil {
```
see https://github.com/kubernetes/kubernetes/pull/106414#issuecomment-972242758

Reply:
kind curious why `savedNode` could be nil though

Reply:
after dig a little bit, found that when there is no`patch` action in actions it will be nil